AppServer/google/appengine/api/taskqueue/taskqueue_distributed.py

```diff
@@ -84,6 +84,8 @@ class TaskQueueServiceStub(apiproxy_stub.APIProxyStub):
   This stub executes tasks when enabled by using the dev_appserver's AddEvent
   capability.
   """
+  _ACCEPTS_REQUEST_ID = True
```

Comment:
The APIProxyStub includes the request ID if the attribute for the relevant stub is `True`: https://github.com/AppScale/appscale/blob/5d23b87368fb41bbb16530f86d25404e595fa1df/AppServer/google/appengine/api/apiproxy_stub.py#L125